Contest_API.md

```diff
@@ -534,6 +528,33 @@ event feed, also note that these should not be considered proper REST
 endpoints, and that the `event-feed` endpoint is a streaming feed in
 NDJSON format.
 
+Note that `access`, `account`, `state`, `scoreboard` and `event-feed`
+are singular nouns and indeed contain only a single object.
+
+### Required and Optional API
+
+The only required endpoints are metadata: `api` and `access`. The
+only required property is `id` (if it exists on the endpoint).
+
+All other endpoints and properties are optional. The only exceptions
+to this are obvious logical requirements (for example, a submission
+must come from a team) and the requirement for referential integrity
```

Comment:
"obvious logical requirements" seems very undefined. 

I'm not even sure I agree that the example is *absolutely* necessary.

Reply:
I agree, but I couldn't come up with a better term that would be any more specific or meaningful. This is really just a weak "it would be silly to do X". Do we just remove this entirely and the only requirements are ids and referential integrity?

Reply:
I would suggest that we just remove this entirely.

Reply:
I'm fine with removing this, but also see my remark above about relating data in `access` to requiring those endpoints and properties.

Reply:
Removed entirely while responding to comment above. /access suggestion was accepted. Please re-review.